doc/source/whatsnew/v0.25.0.rst

```diff
@@ -132,6 +132,7 @@ Other Enhancements
 - :class:`DatetimeIndex` and :class:`TimedeltaIndex` now have a ``mean`` method (:issue:`24757`)
 - :meth:`DataFrame.describe` now formats integer percentiles without decimal point (:issue:`26660`)
 - Added support for reading SPSS .sav files using :func:`read_spss` (:issue:`26537`)
+- Added new option ``plotting.backend`` to be able to select a plotting backend different that the existing ``matplotlib`` one. Use ``pandas.set_option('plotting.backend', '<backend-module>')`` where ``<backend-module`` is a library implementing the pandas plotting API (:issue:`14130`)
```

Comment:
"that" -> "than"

Reply:
We don't have any alternative engines to list here yet, right?

Reply:
Not at the moment, but that's a good point. Next week once this is merged I'm planning to work with few people to adapt `hvplot`. So we can see that everything is working well, and we can fix anything before 0.25. It may make sense to update this and use `hvplot` as an example when it's ready.

Reply:
Great. I think this should be a prominent new feature if we are able to get either or both of pdvega ready to use it in time for the release.